virttest/utils_disk.py

```diff
@@ -659,39 +693,74 @@ def configure_empty_windows_disk(session, did, size, start="0M",
 
     :param session: session object to guest.
     :param did: disk index which show in 'diskpart list disk'.
-    :param size: partition size. e.g. 500M
+    :param size: the total size of partitions. e.g. 2G
     :param start: partition beginning at start. e.g. 0M
     :param n_partitions: the number of partitions on disk
     :param fstype: filesystem type for the disk.
     :param labeltype: label type for the disk.
     :param timeout: Timeout for cmd execution in seconds.
     :return a list: mount point list for all partitions.
     """
+
+    def _get_availabel_size():
+        """Get the free size of the disk."""
+        disk_script = '_'.join(("disk", ''.join(
```

Comment:
could this script name be more meaningful? like dp_script = list_disk_script.dp directly? this script is used to list guest disk. and due to the script will be deleted once executed, so I think here no need use so complex name. just suggestion.

Reply:
Thanks for your comment, I replace it with  "script" more simple.